in release-0.4 branch make create-cluster-management fails

/kind bug

**What steps did you take and what happened:**
in branch `release-0.4` running `make create-cluster-management` produces the following error:

```unable to recognize "examples/_out/provider-components.yaml": no matches for kind "Certificate" in version "certmanager.k8s.io/v1alpha1"
unable to recognize "examples/_out/provider-components.yaml": no matches for kind "Issuer" in version "certmanager.k8s.io/v1alpha1"
Makefile:310: recipe for target 'create-cluster-management' failed
make: *** [create-cluster-management] Error 1
```

**What did you expect to happen:**
`make create-cluster-management` to successfully create a development Kubernetes cluster on AWS in a KIND management cluster.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- Cluster-api-provider-aws version: release-0.4
- Kubernetes version: (use `kubectl version`):  `Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.1", GitCommit:"d647ddbd755faf07169599a625faf302ffc34458", GitTreeState:"clean", BuildDate:"2019-10-02T17:01:15Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}`
- OS (e.g. from `/etc/os-release`): ubuntu

@tahsinrahman thanks, the `CAPI_BRANCH` was not set.

I did a git pull and this is what I'm encountering now:

```
I1101 11:28:29.532937   10904 clusterclient.go:242] Waiting to acquire client... server not yet available: Get https://test1-apiserver-XXXXXXXXX.eu-west-2.elb.amazonaws.com:6443/api?timeout=32s: EOF
F1101 11:28:29.533047   10904 alpha_phase_apply_addons.go:51] unable to create cluster client: failed to acquire new client: timed out waiting for the condition
Makefile:310: recipe for target 'create-cluster-management' failed
```

any idea?

--- a/examples/controlplane/controlplane.yaml
+++ b/examples/controlplane/controlplane.yaml
@@ -23,8 +23,6 @@ metadata:
   name: ${CLUSTER_NAME}-controlplane-0
 spec:
   instanceType: ${CONTROL_PLANE_MACHINE_TYPE}
-  ami:
-    id: ${IMAGE_ID}
   iamInstanceProfile: "control-plane.cluster-api-provider-aws.sigs.k8s.io"
   sshKeyName: "${SSH_KEY_NAME}"
 ---
@@ -71,8 +69,6 @@ metadata:
   name: ${CLUSTER_NAME}-controlplane-1
 spec:
   instanceType: ${CONTROL_PLANE_MACHINE_TYPE}
-  ami:
-    id: ${IMAGE_ID}
   iamInstanceProfile: "control-plane.cluster-api-provider-aws.sigs.k8s.io"
   sshKeyName: "${SSH_KEY_NAME}"
 ---
@@ -113,8 +109,6 @@ metadata:
   name: ${CLUSTER_NAME}-controlplane-2
 spec:
   instanceType: ${CONTROL_PLANE_MACHINE_TYPE}
-  ami:
-    id: ${IMAGE_ID}
   iamInstanceProfile: "control-plane.cluster-api-provider-aws.sigs.k8s.io"
   sshKeyName: "${SSH_KEY_NAME}"
 ---

--- a/examples/controlplane/kustomizeversions.yaml
+++ b/examples/controlplane/kustomizeversions.yaml
@@ -209,3 +209,27 @@ spec:
         echo "kubelet version: " $(kubelet --version)
 
         echo "$LINE_SEPARATOR"
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: AWSMachine
+metadata:
+  name: ${CLUSTER_NAME}-controlplane-0
+spec:
+  ami:
+    id: ${IMAGE_ID}
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: AWSMachine
+metadata:
+  name: ${CLUSTER_NAME}-controlplane-1
+spec:
+  ami:
+    id: ${IMAGE_ID}
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: AWSMachine
+metadata:
+  name: ${CLUSTER_NAME}-controlplane-2
+spec:
+  ami:
+    id: ${IMAGE_ID}

--- a/examples/machinedeployment/kustomizeversions.yaml
+++ b/examples/machinedeployment/kustomizeversions.yaml
@@ -69,3 +69,13 @@ spec:
           echo "kubelet version: " $(kubelet --version)
 
           echo "$LINE_SEPARATOR"
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: AWSMachineTemplate
+metadata:
+  name: ${CLUSTER_NAME}-md-0
+spec:
+  template:
+    spec:
+      ami:
+        id: ${IMAGE_ID}

--- a/examples/machinedeployment/machinedeployment.yaml
+++ b/examples/machinedeployment/machinedeployment.yaml
@@ -36,8 +36,6 @@ spec:
   template:
     spec:
       instanceType: ${NODE_MACHINE_TYPE}
-      ami:
-        id: ${IMAGE_ID}
       iamInstanceProfile: "nodes.cluster-api-provider-aws.sigs.k8s.io"
       sshKeyName: "${SSH_KEY_NAME}"
 ---

